### ISSUE TITLE
Megascript Steam Crashes After Validation, Does Not Install Properly

### ISSUE THREAD

**Issue by tiredNmodding** [cobalt2727/L4T-Megascript#146]
The L4T Megascript pre-prepared installation of Steam (utilizing Box86 and Box64) no longer installs properly, crashing after a "validating Steam" message, with no further progress. Multiple troubleshooting methods have been carried out on my end using the terminal output to look for clues, but to no avail. Oddly enough, this issue has somehow carried over to older Switchroot versions that once did not exhibit this issue, even on clean flashes/installations with no modifications or alterations. It also seems that I am not the only person who has encountered this, and it is at the very least consistently occurring no matter what steps are taken to mitigate possible failure points. Any help that you could give in this regard would be greatly appreciated.  I'll attach a .txt file with the output for when I try to launch Steam through the terminal. Again, I would like to state that this is occurring with the latest versions and clean installations/flashes of both Ubuntu 18.04 Bionic and Ubuntu 22.04 Jammy.

[steam error.txt](https://github.com/cobalt2727/L4T-Megascript/files/13945250/steam.error.txt)

**Comment by cobalt2727** [comment]
@theofficialgman would know better since I haven't really worked with it, but this is _probably_ [a box86 bug](https://github.com/ptitSeb/box86/issues/920)?

**Comment by cobalt2727** [comment]
especially considering you said this happens on both 18.04 and 22.04. doesn't sound like there's much we can do from this end.

**Comment by tiredNmodding** [comment]
Thanks for replying. Honestly, I have no idea; I'm kind of in over my head with this. I have tried building box 86 and 64 for L4T rather than going for the versions that comes with the Steam installer, but the same issue pops up no matter which versions of 86/64 I use. From "research" I was able to do, I know GLX is something having to do with translation between OpenGL and X Window, but that's pretty much it. Anyway, thanks for your time; I'll see if I can post this to the box86/box64 github page and see if there's something that can be done.
